tests/unit/recommenders/evaluation/test_python_evaluation.py

```diff
@@ -194,7 +181,7 @@ def test_python_exp_var(rating_true, rating_pred):
         rating_pred=rating_true,
         col_prediction=DEFAULT_RATING_COL,
     ) == pytest.approx(1.0, TOL)
-    assert exp_var(rating_true, rating_pred) == pytest.approx(-6.4466, TOL)
+    assert exp_var(rating_true, rating_pred) == pytest.approx(-6.4466, 0.01)
```

Comment:
Does this mean we get something like -6.38 or -6.51? It looks a bit strange, since nothing changed with explained variance, doesn't it?

Reply:
I think this happened when I copied expected outputs values from the fixture over to each tests. This should be not changed. I'll update. Good catch!